composer.json

```diff
@@ -31,40 +31,60 @@
             "email": "guilhermeblanco@hotmail.com"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/webimpress/DoctrineModule"
+        }
+    ],
     "require":           {
-        "php":                               "^5.5 || ^7.0",
-        "doctrine/doctrine-module":          "~1.0",
+        "php":                               "^5.6 || ^7.0",
+        "doctrine/doctrine-module":          "dev-feature/zf3-support as 2.0",
         "doctrine/orm":                      ">=2.5,<2.7",
         "doctrine/dbal":                     ">=2.4,<2.7",
-        "zendframework/zend-stdlib":         "^2.7.6",
-        "zendframework/zend-hydrator":       "^1.1",
-        "zendframework/zend-mvc":            "^2.5.2",
-        "zendframework/zend-servicemanager": "^2.7.5 || ^3.0.3",
-        "symfony/console":                   "~2.5|~3.0"
+        "symfony/console":                   "^2.5 || ^3.0",
+        "zendframework/zend-mvc":            "^3.0.2",
+        "zendframework/zend-servicemanager": "^3.1.0",
+        "zendframework/zend-stdlib":         "^3.0.1"
     },
     "require-dev":       {
-        "zendframework/zend-modulemanager":   "^2.6.1",
-        "zendframework/zend-config":          "^2.6",
-        "zendframework/zend-view":            "^2.5.2",
-        "zendframework/zend-serializer":      "^2.6",
-        "zendframework/zend-log":             "^2.5.2",
-        "zendframework/zend-i18n":            "^2.6",
-        "zendframework/zend-console":         "^2.6",
-        "zendframework/zend-code":            "^2.5.2",
-        "doctrine/data-fixtures":             "1.0.*",
-        "zendframework/zend-developer-tools": "^1.0",
-        "doctrine/migrations":                "1.0.*@dev",
-        "phpunit/phpunit":                    "~4.0",
-        "squizlabs/php_codesniffer":          "~2"
+        "doctrine/data-fixtures":             "^1.2",
+        "doctrine/migrations":                "^1.4",
+        "phpunit/phpunit":                    "^5.4",
+        "squizlabs/php_codesniffer":          "^2.6.2",
+        "zendframework/zend-developer-tools": "^1.1",
+        "zendframework/zend-i18n":            "^2.7.3",
+        "zendframework/zend-modulemanager":   "^2.7.2"
     },
     "suggest":           {
         "zendframework/zend-form":            "if you want to use form elements backed by Doctrine",
         "zendframework/zend-developer-tools": "zend-developer-tools if you want to profile operations executed by the ORM during development",
         "doctrine/migrations":                "doctrine migrations if you want to keep your schema definitions versioned"
     },
     "autoload":          {
-        "psr-0": {
+        "psr-4": {
```

Comment:
@Maaslos-UG 

Switching from PSR-0 to PSR-0 could be addressed in a separate PR, too.

What do you think?